File: RolePlayGame/java/fixtures/my/examples/CalculatorOneFixture.java
package my.examples;

import fit.ColumnFixture;

public class CalculatorOneFixture extends ColumnFixture {
	public double numerator;
	public double denominator;
	
	public double quotient(){
		Calculator calculator = new Calculator(); // system under test
		return calculator.quotientFor(numerator, denominator);
	}
}

File: RolePlayGame/dotnet/fixtures/my.examples/CalculatorOneFixture.cs
using fit;

namespace my.examples {

	public class CalculatorOneFixture : ColumnFixture {
		public double numerator;
		public double denominator;
		
		public double Quotient(){
			Calculator calculator = new Calculator(); // system under test
			return calculator.QuotientFor(numerator, denominator);
		}
	}
}